### DISCUSSION TITLE
Official AITemplate support (potentially significant speedup)

### DISCUSSION THREAD

**Discussion by Dampfinchen** [comfyanonymous/ComfyUI#2620] [Ideas]
Hello, 

I've been hearing good things about a significant speedup using AITemplate so I wanted to install it. However, the latest repo https://github.com/FizzleDorf/ComfyUI-AIT (which is 2 months old at this point) doesn't seem to be maintained anymore and thus, doesn't work with the latest versions of Comfy UI. 

Any chance we can get official support? It seems to be a good alternative to tensorRT as it doesn't require you to convert models and such.

**Comment by Meizr** [comment]
+1 on this.

Never got that to work here though.

But the fact you don't need to convert the models is a great advantage.
